Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml

```diff
@@ -75,38 +115,64 @@
   physicalDesc: opaque
   color: "#DFDFDF"
   plantMetabolism:
-    - !type:AdjustNutrition
+    - !type:PlantAdjustNutrition
       amount: 0.1
-    - !type:AdjustWater
+    - !type:PlantAdjustWater
       amount: 0.9
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 4
 
 - type: reagent
   id: MilkOat
   name: oat milk
   desc: Surprisingly tasty.
   physicalDesc: refreshing
   color: "#302000"
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 4
 
 - type: reagent
   id: MilkSoy
   name: soy milk
   desc: Surprisingly tasty.
   physicalDesc: refreshing
   color: "#302000"
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 5 # soyboys stay winning
 
 - type: reagent
   id: MilkSpoiled
   name: spoiled milk
   desc: This milk has gone rancid.
   physicalDesc: putrid
   color: "#faffba"
+  metabolisms:
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: -2
```

Comment:
I beg you

Reply:
well theres not much i can do about this one since i do have to specify a different value